app/admin/top-issues/components/AdminTopIssuesPage.js

```diff
@@ -35,14 +38,16 @@ export default function AdminTopIssuesPage(props) {
         isError: false,
       };
     });
-    await createTopIssue(topIssueName);
+    setTopIssues([
+      await createTopIssue(topIssueName, svgData),
+      ...topIssues
+    ])
```

Comment:
```
const newIssue = await createTopIssue(topIssueName, svgData);
setTopIssues([
  ...topIssues,
  newIssue,
    ])
```

Reply:
Why do you want me to reverse the spreading?  Wouldn't that place the newly added issue at the end of the list, potentially far below the page fold? 🤔

Reply:
oh I didn't realized the order was important there.
I usually spread first so the new one will override the old one. If the order matters here, keep it as it was.